packages/node_modules/@webex/internal-plugin-user/src/user.js

```diff
@@ -288,6 +288,87 @@ const User = WebexPlugin.extend({
   },
 
   /**
+   * Determines if the specified user needs to signup or can signin.
+   * Triggers activation email if client credentials are used.
+   * @param {Object} params
+   * @param {string} params.email (required)
+   * @returns {Promise<Object>}
+   */
+  validate(params) {
+    return new Promise((resolve, reject) => {
+      // merge options
+      const options = {...this.config.verifyDefaults, ...params};
+
+      // validate options has email key value
+      if (!options.email) {
+        reject(new Error('`options.email` is required'));
+      }
+
+      // destructure for ease of access
+      const {services} = this.webex.internal;
+
+      // scoped globals
+      let authToken;
+
+      services.updateServices('limited', 'email', options.email)
+        // retrieved an error when updating limited services
+        .catch((e) => {
+          e.message = ['failed to update limited catalog: ', e.message]
+            .join('');
+          reject(e);
+        })
+        // successfully updated limited service catalog
+        .then(() => this.webex.credentials.getUserToken())
```

Comment:
Is this a short circuit if we already have the credentials loaded from local storage? According to the [wiki](https://wiki.cisco.com/pages/viewpage.action?pageId=264251595) I thought we always had to hit the /signing catalog with a client token and that would return the user details or 404